extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -106,10 +106,7 @@ export class AmpSelector extends AMP.BaseElement {
     this.init_();
     if (!this.isDisabled_) {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
-      if (this.kbSelectMode_ != KEYBOARD_SELECT_MODES.NONE) {
```

Comment:
we definitely dont need this on mobile devices, seems to be unnecessary when some one does not set keyboard option